tests/h/routes_test.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from mock import Mock, call
+
+from h.routes import includeme
+
+
+def test_includeme():
+    config = Mock(spec_set=['add_route'])
+
+    includeme(config)
+
+    # This may look like a ridiculous test, but the cost of keeping it
```

Comment:
> This may look like a ridiculous test

My thoughts exactly, although I suppose it does at least make sure that the code runs without error.